runtimebp/config.go

```diff
@@ -20,21 +18,17 @@ type Config struct {
 	} `yaml:"numProcesses"`
 }
 
-// InitFromConfig sets GOMAXPROCS using the given config.
+// InitFromConfig configures the runtime's GOMAXPROCS using the following heuristic:
+//   1. If $GOMAXPROCS is set, it relinquishes control to the Go runtime.
+//      This should cause the runtime to respect this value directly.
+//   2. If $BASEPLATE_CPU_REQUEST is unset/invalid, it relinquishes control to automaxprocs, minimum 2.
+//      See https://pkg.go.dev/go.uber.org/automaxprocs for specific behavior.
+//   3. Otherwise, $BASEPLATE_CPU_REQUEST is multiplied by $BASEPLATE_CPU_REQUEST_SCALE
+//      (or defaultCPURequestScale) to compute the new GOMAXPROCS, minimum 2.
```

Comment:
nit: this will be fixed in go 1.19, but for now we do need blank lines between the ordered list items to have them rendered correctly on pkgsite. we also need to unindent them otherwise they'll be rendered as blockquotes.

Reply:
Thanks!